src/skai/generate_examples.py

```diff
@@ -374,7 +374,7 @@ def _create_example(
       before_image: np.ndarray,
       after_image_id: str,
       after_image: np.ndarray,
-      scalar_features: Dict[str, List[float]]) -> Optional[Example]:
```

Comment:
Can you make this a List[Union[float, str]] so that it's clear what the possible values are?

Reply:
[3803eb0](https://github.com/google-research/skai/pull/89/commits/3803eb0a93b940e84fa9715193c17d52cd451bce)
Additional changes to encode List[str] for [string_label]